### DISCUSSION TITLE
Connecting several BMS to one USB <-> RS485 adapter

### DISCUSSION THREAD

**Discussion by thorstenpawlak** [Louisvdw/dbus-serialbattery#112] [Q&A] [answered]
Hi,
I already have an USB-RS485 Adapter with an EM24 connected, can i connect 2 JK-BMS to that? Victron sais that several EM24 work in parallel at one adapter if the adresses are changed.

Thanks and regards

**Comment by Louisvdw** [comment, marked as answer]
Hi
That only works with the EM24 and other Carlo meters. There is a tool you run that you can set each meter's address. 
Most BMS does not have this functionality and the serial driver does not handle anything like that. You need a new USB-RS485 for each BMS

**Reply by thorstenpawlak** [reply]
Thanks, do you recommend an USB-RS485 Adapter? The one from Victron is quiet expensive.

**Reply by Louisvdw** [reply]
Anyone based on the FTDI chip should work well. 
I know this one works 
![image](https://user-images.githubusercontent.com/26077665/160755508-4491452a-99d6-4cf2-8df5-18a237046d46.png)
And the Wave USB to RS485 also works
